backoffice/backoffice-internal-domain-service/ballerina/backoffice_internal_dao.bal

```diff
@@ -0,0 +1,24 @@
+import ballerinax/postgresql;
+import ballerina/sql;
+
+
+public function getConnection() returns postgresql:Client | error {
+    //Todo: Need to read database config from toml
+    postgresql:Client|sql:Error dbClient = 
+                                check new ("localhost", "admin", "admin", 
+                                     "APKDB", 5432);
+    return dbClient;   
+}
+
+public function db_createAPI(API api) returns sql:ExecutionResult | sql:Error{
+    postgresql:Client | error db_client  = getConnection();
+    if db_client is error {
+        return error("Issue while conecting to databse");
+    } else {
+        //Todo: query need to improve
+        sql:ParameterizedQuery query = `INSERT INTO am_api(api_name, api_version,context,api_provider,status,artifact)
```

Comment:
Isn't it better to maintain queries in a seperate bal file?

Reply:
Fix https://github.com/wso2/apk/pull/276/commits/830d6993e6e9f0a867caed16b80c2aadef75a409